packages/auth/src/accessToken/service.ts

```diff
@@ -8,17 +8,25 @@ import { ClientService } from '../client/service'
 import { AccessToken } from './model'
 import { IAppConfig } from '../config/app'
 
+interface RevokeTokenArgs {
+  managementId: string
+  tokenValue: string
+  trx?: TransactionOrKnex
+}
+
 interface RotateTokenArgs {
+  grantId: string
   managementId: string
   tokenValue: string
+  expiresIn?: number
   trx?: TransactionOrKnex
 }
 
 export interface AccessTokenService {
   get(token: string): Promise<AccessToken | undefined>
   getByManagementId(managementId: string): Promise<AccessToken | undefined>
   introspect(token: string): Promise<Grant | undefined>
-  revoke(id: string, tokenValue: string): Promise<void>
+  revoke(args: RevokeTokenArgs): Promise<boolean>
   create(grantId: string, opts?: AccessTokenOpts): Promise<AccessToken>
   rotate(args: RotateTokenArgs): Promise<AccessToken | undefined>
```

Comment:
```suggestion
  revoke(id: string, trx?: TransactionOrKnex): Promise<boolean>
  create(grantId: string, opts?: AccessTokenOpts): Promise<AccessToken>
  rotate(id: string, trx?: TransactionOrKnex): Promise<AccessToken | undefined>
```
We have the tokens

Reply:
I still needed `grantId` for the `rotate` but made the suggested changed

Reply:
I still needed `grantId` for the `rotate` but made the suggested changed